tmt/base.py

```diff
@@ -948,6 +948,32 @@ def export(self, format_='rst', title=True):
 
         return output
 
+    def _lint_summary(self):
+        """ Lint summary step """
+        # Summary is advised with a resonable length
+        if self.summary is None:
+            verdict(None, 'summary is very useful for quick inspection')
+        elif len(self.summary) > 50:
+            verdict(None, 'summary should not exceed 50 characters')
+        else:
+            verdict(True, "summary is ok")
```

Comment:
Yeah, good idea. Actually we already have the `Node` class which covers common stuff across `Test`, `Plan` and `Story`. Let's share the `summary` check there. It might also make sense to rename `Node` to `Core` in the future to make this a bit more obvious.

Reply:
Oh, I see now. The name `Node` confused me at first, I thought it was referring to an fmf node which may not have `summary` attribute set but looking at the code now, it indeed represents the core metadata specification. @adka1408 could you do the changes in this PR? Should be fairly simple, I'd rename `Node` to`Core` and replace the 3 identical summary checks with a method in the `Core` class?

Reply:
Just pondering, renaming the class is a backward-incompatible change. I **guess** there are just a few people/teams who use `tmt` directly from Python (remember @t184256 was processing test metadata) but still... Perhaps we should do the rename in `tmt-2.0`? @lukaszachy, @FrNecas, @thrix, what do you think?

Reply:
We can add `Node = Core` into base.py until the EOL of 1.x releases.

Reply:
I'd prefer to have the class renamed ASAP for clarity as we may forget about it if we decide to only change it in 2.0. So IMHO renaming and adding an "alias" as Lukas suggested makes sense to me.

Reply:
Sounds good to rename the class soon and provide `Node = Core` for backward-compatibility.